openshift_sti_images/nodejs.adoc

```diff
@@ -0,0 +1,130 @@
+= NodeJS
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Our OpenShift https://github.com/openshift/sti-nodejs/tree/master/0.10[NodeJS] image repository contains the sources and Dockerfiles for building various versions of NodeJS platform images. The resulting images can be run either by Docker or using https://github.com/openshift/source-to-image[STI].
+
+== Versions
+Current supported versions of NodeJS:
+
+* https://github.com/openshift/sti-nodejs/tree/master/0.10[0.10]
+
+== Base Images
+
+* RHEL-7
+* CentOS-7
+
+[NOTE]
+RHEL-7 image is not available as trusted build in https://index.docker.io[Docker Index]. In order to perform build or test action on a RHEL-7 based NodaJS image, you need to run the build on properly subscribed RHEL-7 machine. 
+
+== Installation
+To build NodeJS image, choose between CentOS7 or RHEL7 base image:
+
+==== RHEL-7 base image
+
+----
+$ git clone https://github.com/openshift/sti-nodejs.git
+$ cd sti-nodejs
+$ make build TARGET=rhel7 VERSION=0.10
+----
+
+==== CentOS-7 base image
+
+----
+$ git clone https://github.com/openshift/sti-nodejs.git
+$ cd sti-nodejs
+$ make build VERSION=0.10
+----
+
+[NOTE]
+By omitting the `VERSION` parameter, the build/test action will be performed on all the supported versions of NodeJS. Since we are now supporting only version `0.10`, you can omit this parameter
+
+== Usage
+Building simple https://github.com/ryanj/node-echo[nodejs-echo-app] NodeJS application, using standalone https://github.com/openshift/source-to-image[STI] and running the resulting image by http://docker.io[Docker]
+
+==== RHEL-7 base image
+
+----
+$ sti build https://github.com/ryanj/node-echo.git nodejs-0.10-rhel7 nodejs-echo-app
+$ docker run -p 3000:3000 nodejs-echo-app
+----
+
+==== CentOS-7 base image
+
+----
+$ sti build https://github.com/ryanj/node-echo.git nodejs-0.10-centos7 nodejs-echo-app
+$ docker run -p 3000:3000 nodejs-echo-app
+----
+
+To access and test if the application is running run:
+----
+$ curl 127.0.0.1:3000
+----
+
+== Test
+OpenShift https://github.com/openshift/sti-nodejs/tree/master/0.10[STI-NodeJS] repository also provides STI test framework, which launches test to check functionality of a simple nodejs application built on top of sti-nodejs image.
+
+==== RHEL-7 base image
+
+----
+$ cd sti-nodejs
+$ make test TARGET=rhel7 VERSION=0.10
+----
+
+==== CentOS-7 base image
+
+----
+$ cd sti-nodejs
+$ make test VERSION=0.10
+----
+
+[NOTE]
+By omitting the `VERSION` parameter, the build/test action will be performed on all the supported versions of NodeJS. Since we are now supporting only version `0.10`, you can omit this parameter
+
+
+== Repository organization
+
+* `/nodejs-version/`
```

Comment:
Please don't put whitespace before nested lists. In AsciiDoc the pattern is:

    * Level 1
    ** Level 2
    *** Level 3